apps/meteor/app/lib/server/lib/sendNotificationsOnMessage.js

```diff
@@ -368,6 +369,10 @@ export async function sendAllNotifications(message, room) {
 			mentions.map(async (userId) => {
 				await callJoinRoom(userId, room._id);
 
+				if (room.teamId) {
+					Team.addMember(room.u._id, userId, room.teamId);
+				}
+
```

Comment:
I think a better option would be to change the `addUserToRoom` function instead of adding the team member here. This way, we'd also cover the case where a user joins a team using the "Join" button (I've just noticed we also don't create a new team member in this case)

Reply:
Updated!